src/custom/components/SwapWarnings/index.tsx

```diff
@@ -104,14 +105,31 @@ const HighFeeWarningMessage = ({ feePercentage }: { feePercentage?: Fraction })
   </div>
 )
 
-export type HighFeeWarningProps = {
+const NoImpactWarningMessage = (
+  <div>
+    <small>
+      We are unfortunately unable to calculate any price impact for this trade.
+      <br />
+      <br />
+      <u>
+        <strong>We strongly advise you to do your due diligence before advancing.</strong>
+      </u>
+      <br />
+      <br />
+      You may still move forward with this order but there is a potentially high risk of executing an unfavorable trade.
```

Comment:
For me this message is too alarmist. I think i suggested in some other PR something softer, but don't remember in which one

Reply:
"You may still move forward with this order but make sure the receive amounts are what you expect."

Reply:
done 9cc6852